<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPowerForm</class>
 <widget class="QWidget" name="AddPowerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
font: 11pt &quot;微软雅黑&quot;;
background-color: #0f1228;
color: rgb(255, 255, 255);
}
.QPushButton{
font: 9pt &quot;微软雅黑&quot;;
border-radius:3px;
border-style:none;
	color: rgb(255, 255, 255);
background-color: #005AFF;
}
.QPushButton:hover{
padding:4px 4px 2px 4px;
background-color: #005AFF;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#192755;</string>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/image/close.png);
}
QPushButton:hover{
	border-image: url(:/image/close_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res.qrc">:/image/battery_discharge.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>5</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>增加设备</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>串口信息</string>
   </property>
   <widget class="QComboBox" name="cbt_parity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>none</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>odd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mark</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbt_com">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbt_stopbits">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>one</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>two</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>onepointfive</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbt_databits">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbt_baudrate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶校验</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_com_alarm">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>设备信息</string>
   </property>
   <widget class="QLineEdit" name="led_site">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设备类型</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BMS_1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设备ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_power_id">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>安装位置</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_sure">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #005AFF;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
